File: backend/src/models/packagestation.js
import mongoose from "mongoose";

//Schema for a Packagestaion
const packagestationSchema = new mongoose.Schema({
  number: String,
  street: String,
  city: String,
  zip: String,
  country: String,
  status: String,
});

export const Packagestation = mongoose.model(
  "Packagestation",
  packagestationSchema
);

File: backend/models/packagestation.go
package models

import (
	"gorm.io/gorm"
)

type Packagestation struct {
	gorm.Model
	Number  string `json:"number"`
	Street  string `json:"street"`
	City    string `json:"city"`
	Zip     string `json:"zip"`
	Country string `json:"country"`
	Status  string `json:"status"`
}
